### PR TITLE
Implemented Discord Bot

### PR DESCRIPTION
**Pull request by hopingsteam**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
+    "activities_limit": 162,
     "strava_access_token": "<INSERT_ACCESS_TOKEN>",
     "strava_club_id": "<INSERT_CLUB_ID>",
-    "activities_limit": 162
+    "discord_token": "<INSERT_DISCORD_TOKEN>"
```
Comment (CiprianAmz) [inline config.json:5]: What if the user wants to use just the Strava tool? Can they do it without providing discord info?

File: logs/tomi.log [hunk 2]
```diff
@@ -0,0 +1 @@
+Hello, world!
```
Comment (CiprianAmz) [inline logs/tomi.log:1]: Leftover?

File: config.json (hunk 1 above)
Comment (CiprianAmz) [reply config.json:5]: Checked downside and it's fine since you are calling just the Strava script

**Review by CiprianAmz — CHANGES_REQUESTED** [review]
- No connection with Strava & the bot.

I suggest the following:
- Reshaping the package to have VCS Support. That will allow installing the tool using pip (example: https://stackoverflow.com/questions/20101834/pip-install-from-git-repo-branch/20101940#20101940) More info on how can be done here:
    - https://pip.pypa.io/en/stable/topics/vcs-support/
    - https://docs.python-guide.org/writing/structure/
- Making a project and different repo for Tomatul Biciclistu which will consume the Strava app.

File: src/vendors/discords.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import random
+
+from discord import Intents, Client
+
+from src.utils.logs import tomi_logger
+
+owner_id = 279996271388000256 # Maurice
+
+class TomitaBiciclistul(Client):
```
[the remark this answers is not in the document]
Comment (hopingsteam) [reply src/vendors/discords.py:9]: It doesn't work, the connection will be made in the next PR.

File: src/vendors/discords.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import random
+
+from discord import Intents, Client
+
+from src.utils.logs import tomi_logger
+
+owner_id = 279996271388000256 # Maurice
```
[the remark this answers is not in the document]
Comment (hopingsteam) [reply src/vendors/discords.py:7]: I will move it as an argument, then.

**Pull request merged**